backend/src/main/java/com/mcmasterbaja/analyzer/LinearMultiplyAnalyzer.java

```diff
@@ -37,7 +37,7 @@ public void analyze() {
     Reader r = new CSVReader(super.inputFiles[0]);
     Writer w = new CSVWriter(super.outputFiles[0]);
 
-    w.write(linearMultiply(r.read(), m, b));
+    w.write(linearMultiply(r.read(), m, b)); 
```

Comment:
try to avoid making changes like this to random files, clutters the PR

Reply:
This convo has not been resolved